checkClock/Gemfile

```diff
@@ -56,3 +56,5 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'devise'
+
+gem 'dotenv', '~> 2.1'
```

Comment:
Pls, move this to the dev and test gems group
It's not recommended to use it in production envs

Reply:
Done